scripts/prompt-config.sh

```diff
@@ -886,7 +887,7 @@ export TERM=xterm-color
 
 trap sig_doshell SIGINT
 
-printheader "Copyright 2025 MNX Cloud, Inc."
+printheader "Copyright 2025 Edgecast Cloud LLC."
```

Comment:
This string should be "Edgecast Cloud LLC (edgecast.io)", right?.

Reply:
No.  That format is typically only used in `package.json` files.